internal/modules/authz/test.py

```diff
@@ -0,0 +1,106 @@
+import os
+import time, logging
+import tempfile
+import unittest
+from core.keys import keys_show
+from modules.bank.query import (
+    query_balances,
+)
+from modules.authz.tx import *
+from modules.authz.query import *
+
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
+
+# get account addresses
+granter = keys_show("account1")[1]["address"]
+grantee = keys_show("account2")[1]["address"]
+receiver = keys_show("validator1")[1]["address"]
+amount = 5
+
+temp = tempfile.TemporaryFile()
+temp_file = f"{temp.name}.json"
+
+
+class TestAuthzModuleTxsQueries(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # grant tx
+        status, grant = tx_grant_authz(granter, grantee)
+        assert status, "error in authz grant tx!!!"
+        time.sleep(3)
+
+    def test_authz_grant(self):
+        status, grants = query_authz_grants(granter, grantee)
+        self.assertTrue(status)
+        spend_limit = grants["grants"][0]["authorization"]["spend_limit"][0]["amount"]
```

Comment:
should check for grants length before testing this. It might break the tests if grants are nil